panellayout/build.gradle

```diff
@@ -17,6 +17,10 @@ android {
     kotlinOptions {
         jvmTarget = JavaVersion.VERSION_1_8
     }
+    buildFeatures {
+        buildConfig = false
+        resValues = false
```

Comment:
What are these for?

Just being curious..

Reply:
We have these disabled also in our project at Wayfair. They are Android Gradle Plugin features that we don't need. It is actually better for libraries to disable these since `BuildConfig` will be leaked into consumers otherwise. 

Last year, Jake Wharton did this with the same commit in many many projects of his and Square.